explainer.md

```diff
@@ -187,6 +187,30 @@ There are several open questions to address:
    source.
 1. How to handle messaging.
 
+## Security and privacy considerations
+
+See [the corresponding section](
+https://w3c.github.io/presentation-api/#security-and-privacy-considerations) of
+the Presentation API spec. Below, we list characteristics unique to site
+initiated mirroring.
+
+### Personally identifiable information
+
+By [monitoring the list of available presentation displays](
+https://w3c.github.io/presentation-api/#monitoring-the-list-of-available-presentation-displays),
+a website can determine whether the user has devices compatible with mirroring,
```

Comment:
Let's use "presentation displays" instead of "devices" for consistency with the spec.